### ISSUE TITLE
projectaria-tools incompatible with project projectaria-client-sdk

### ISSUE THREAD

**Issue by ReinderVosDeWael** [facebookresearch/projectaria_tools#89]
Hi,

I'm attempting to install `proiectaria-tools` into a project that already has `projectaria-client-sdk` as a dependency and get the following error: 

```shell
aria-tests git:(main) ✗ poetry add 'projectaria-tools[all]' 
Using version ^1.5.0 for projectaria-tools

Updating dependencies
Resolving dependencies... (0.7s)

Because no versions of projectaria-tools match >1.5.0,<2.0.0
 and projectaria-tools[all] (1.5.0) depends on projectaria-tools (1.5.0), projectaria-tools[all] (>=1.5.0,<2.0.0) requires projectaria-tools (1.5.0).
And because projectaria-client-sdk (1.0.3) depends on projectaria-tools (1.2.0)
 and no versions of projectaria-client-sdk match >1.0.3,<2.0.0, projectaria-tools[all] (>=1.5.0,<2.0.0) is incompatible with projectaria-client-sdk (>=1.0.3,<2.0.0).
So, because aria-tests depends on both projectaria-client-sdk (^1.0.3) and projectaria-tools[all] (^1.5.0), version solving failed.
```

Locking projectaria-tools to 1.2.0 does resolve the issue, but it seems odd that the latest versions are incompatible.

**Comment by SeaOtocinclus** [comment]
Hi, Can you try to force update the package and tell us if that works?
`python3 -m pip install projectaria-tools --upgrade`

**Comment by ReinderVosDeWael** [comment]
Hi, this will indeed force through the installation and resolves the issue within my current use-case. 

That being said, it'd be great if the underlying issue could be resolved as this prevents the usage of stricter environment management tools like Poetry.

**Comment by SeaOtocinclus** [comment]
Closing the issue, thank you for your feedback
